package.json

```diff
@@ -12,6 +12,7 @@
   "license": "AGPLv3",
   "dependencies": {
     "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
+    "esn-frontend-mailto-handler": "github:openpaas-suite/esn-frontend-mailto-handler#main",
```

Comment:
@chamerling you need to convert `master` branch to `main` branch for the `mailto-handler` repo

Reply:
OMG, one more time I failed on this. Doing it right now